core/variant/array.h

```diff
@@ -89,6 +89,8 @@ class Array {
 
 	int find(const Variant &p_value, int p_from = 0) const;
 	int rfind(const Variant &p_value, int p_from = -1) const;
+	int find_if(const Callable &p_func, int p_from = 0) const;
```

Comment:
```suggestion
	int find_if(const Callable &p_callable, int p_from = 0) const;
```
Match the other ones taking a callable, keep it `func` in the bindings